packages/dns-discovery/src/dns_over_https.ts

```diff
@@ -16,10 +16,10 @@ export class DnsOverHttps implements DnsClient {
    */
   static DefaultEndpoints: Endpoint[] = [
     toEndpoint({
-      name: "cisco-doh",
+      name: "AhaDNS",
       protocol: "https:",
-      host: "doh.opendns.com",
-      ipv4: "146.112.41.2",
+      host: "https://doh.la.ahadns.net/dns-query",
```

Comment:
```suggestion
      host: "doh.la.ahadns.net",
```
both side are automatically completed by the doH JS lib